src/migration/PoC_dfusion.js

```diff
@@ -1,4 +1,4 @@
-const { isDevelopmentNetwork, getDependency } = require("./migration_utilities.js")
+const { isDevelopmentNetwork, getDependency } = require("./utilities.js")
```

Comment:
Is it valuable to include these in the package?

Reply:
@josojo mentioned in #280 that we wanted the migrations to be usable from the outside, I think adding them to the NPM package is correct way to do this.

Reply:
Just to clarify, they are currently included in the NPM package but not re-exported by `index.js`.

Reply:
Yeah, I think it would be great, if external projects only have to import an npm package and then can use it for internal deployments in ganache.